### ISSUE TITLE
Contact information for potential contribution

### ISSUE THREAD

**Issue by cccs-mog** [hasherezade/hollows_hunter#19]
Hi @hasherezade, it seem your only contact information available are behind paywalls. I would like to discuss regarding a potential PR and the way it could be integrated as well as collaborations. Thank you

**Comment by hasherezade** [comment]
Hi @cccs-mog ! Sure we can talk about it, I always welcome collaborations.
BTW, I have no idea what do you mean by "only contact information available are behind paywalls"? Not a single account of mine is behind a paywall. I am available via X/twitter, Wire, here, and also my emails are publicly known. If you prefer talking via e-mail you can reach me out on one of those: hasherezade-at-protonmail.com or hasherezade-at-gmail.com (in both cases the handle is the same as here).

**Comment by cccs-mog** [comment]
Hi @hasherezade, thank you for your response. I will reach out by email. What I meant by paywall is that Twitter/X require a subscription for direct messaging unless I think you follow them or something, Wire is subscription based only as well. I reached out here since in my knowledge there is no way to direct message someone from github. I did not do extra digging but I couldn't find your email address from your github profile or your website contact information, apologies if I missed it.
